andela/boondocks-bn-backend
# #171080162 Line manager email notification after trip re…

### What does this PR do?
- Line manager email notification after trip request
#### Description of Task to be completed?
- [X] Issue email notification to line manager after trip request created under his direct report
#### How should this be manually tested?
- Git clone this repo 
- Run `npm i`
- Run `npm run watch`
- Create trip request 
- Check requester's line manager email for notification
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#171080162](https://www.pivotaltracker.com/story/show/171080162)
#### Screenshots (if appropriate)

![image](https://user-images.githubusercontent.com/52410156/73927245-b0d3e000-48d9-11ea-9ae0-825ba220ff0e.png)


#### Questions:
